src/test/resources/emails/instructorNewAccountRequestAcknowledgementEmailWithComments.html

```diff
@@ -0,0 +1,66 @@
+<p>Hello, Darth Vader</p>
+
+<p>
+  Thank you for submitting an account request. This is what you have submitted:
+</p>
+
+<div>
+  <table style="max-width:600px;border:1px solid black;">
+    <tr>
+      <td style="padding:5px;">
+        <strong>
+          Full Name
+        </strong>
+      </td>
+      <td style="padding:5px;">
+        Darth Vader
+      </td>
+    </tr>
+
+    <tr>
+      <td style="padding:5px;">
+        <strong>
+          Country & Institute
+        </strong>
+      </td>
+      <td style="padding:5px;">
+        Sith Order
+      </td>
+    </tr>
+
+    <tr>
+      <td style="padding:5px;">
+        <strong>
+          Email Address
+        </strong>
+      </td>
+      <td style="padding:5px;">
+        darth-vader@sith.org
+      </td>
+    </tr>
+
+    <tr>
+      <td style="padding:5px;">
+        <strong>
+          Home Page URL<br>& Comments
+        </strong>
+      </td>
+      <td style="padding:5px;">
+        I Am Your Father
+      </td>
+    </tr>
+  </table>
+</div>
+
+<p>
+  Your request will be reviewed within 24 hours. We will send another email once your request has been accepted.
+</p>
+<p>
+  If you have any additional queries, please feel free to contact us at
+  <a href="mailto:teammates@comp.nus.edu.sg">teammates@comp.nus.edu.sg</a>.
```

Comment:
Let's change this to `${support.email}` so that it stays updated even if the support email address changes. For examples, take a look at the following. I'm also not sure if it needs to be hyperlinked; when I do a search, it seems like it never is.

https://github.com/TEAMMATES/teammates/blob/e51132e3c4f2c694c60ac53e3f4a2dd973f9c1c9/src/test/resources/emails/deadlineExtensionGivenInstructor.html#L89